### PR TITLE
allow pgreba to start when postgres is down

### PR DESCRIPTION
**Pull request by benschinn**
Addresses https://github.com/film42/pgreba/issues/19

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -89,13 +89,14 @@ func main() {
 	}
 	ds, err := NewPgReplicationDataSource(cfg)
```
Comment (film42) [inline main.go]: Is there a way we can make a new pg replication data source not return an error when initialized?

Comment (benschinn) [reply main.go]: I explored a few different options, but just landed on deferring the error at initialization. thoughts?

File: data_source.go [hunk 2]
```diff
@@ -305,42 +342,74 @@ func (ds *pgDataSource) getPgCurrentWalLsn(role string) (string, error) {
...
+	err := db.Get(&byteLag, query)
 	if err != nil {
 		return 0, err
 	}
 	return byteLag, nil
 }
 
 func (ds *pgDataSource) IsInRecovery() (bool, error) {
+	db, dbErr := ds.getDB()
+
```
Comment (film42) [inline data_source.go]: Nit-pick but for styling can we remove this extra line between `dbErr` and `dbErr != nil`. Few other instances in this file.

**Review by film42 — APPROVED** [review]
Woot! Let's do a minor version bump with this.

**Pull request merged**